invenio_app_rdm/static_pages/static_pages.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Static pages."""
+from pathlib import Path
+
+from invenio_db import db
+from invenio_pages import Page
+from invenio_rdm_records.fixtures.fixture import FixtureMixin
+
+
+class StaticPages(FixtureMixin):
+    """Static pages."""
+
+    def __init__(self, search_paths, filename, template_path, force=False):
+        """Initialize the fixture."""
+        super().__init__(search_paths, filename)
+        self._force = force
+        self._template_path = template_path
+
+    def load(self):
+        """Load the static pages."""
+        if self._force:
+            Page.query.delete()
+        super().load()
+
+    def page_data(self, page):
+        """Returns content of template."""
+        for path in self._template_path:
+            filepath = path / page
+
+            if not filepath.exists():
+                continue
+
+            return Path(filepath).read_bytes().decode("utf8")
+
+    def create(self, entry):
+        """Load a single page."""
+
+        db.session.add(
+            Page(
+                url=entry.pop("url"),
+                title=entry.get("title"),
+                description=entry.get("description"),
+                content=self.page_data(entry.get("template")),
+                template_name="invenio_pages/default.html",
```

Comment:
I would probably use a new default template defined here. The reason is that the invenio-pages default is imposing some layouting... For example, I would have a:
```
{% extends 'invenio_pages/default_html' %}

{% set title = page.title %}

{% block page_body %}
{% block content %}{{ page.content|safe }}{% endblock %}
{% endblock %}
```

Reply:
Yes! I already added it in the cookiecutter PR, you can check it [here](https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/235)

Reply:
I think we can just omit the template from cookiecutter and define it here... Then, if someone needs to override it, they can do it in cookiecutter, wdyt?